packages/matchbox/src/components/Table/Table.js

```diff
@@ -74,50 +50,33 @@ function Table(props) {
   ) : (
     children
   );
-
   const { px = '450', py = '400', ...paddingProps } = pick(rest, padding.propNames);
   const marginProps = pick(rest, margin.propNames);
 
   return (
-    <Box position="relative">
-      <TablePaddingContext.Provider value={{ px, py, ...paddingProps }}>
-        {freezeFirstColumn && (
-          <DuplicatedTable
-            data-id="matchbox-sticky-table"
-            position="absolute"
-            top="0"
-            left="0"
-            right="0"
-            overflow="clip visible"
-            isScrolled={isScrolled}
-          >
-            <StyledTable>{dataMarkup}</StyledTable>
-          </DuplicatedTable>
-        )}
-        <Wrapper
-          data-id="matchbox-scroll-wrapper"
-          freezeFirstColumn={freezeFirstColumn}
-          onScroll={freezeFirstColumn ? handleScroll : null}
-          {...marginProps}
-        >
-          <StyledTable
-            aria-readonly={readOnly}
-            data-id={dataId}
-            freezeFirstColumn={freezeFirstColumn}
-            id={id}
-            isScrolled={isScrolled}
-            role={role}
-          >
-            {title && (
-              <caption>
-                <ScreenReaderOnly>{title}</ScreenReaderOnly>
-              </caption>
-            )}
-            {dataMarkup}
-          </StyledTable>
-        </Wrapper>
-      </TablePaddingContext.Provider>
-    </Box>
+    <Wrapper
+      freezeFirstColumn={freezeFirstColumn}
+      onScroll={freezeFirstColumn ? handleScroll : null}
+      {...marginProps}
+    >
+      <StyledTable
+        aria-readonly={readOnly}
+        data-id={dataId}
+        freezeFirstColumn={freezeFirstColumn}
+        id={id}
+        isScrolled={isScrolled}
```

Comment:
There is a bug in the current version of chrome that prevents box shadow from rendering I think. Looks fine on other browsers.

Reply:
I think you'd have to use display: block or inline-block to get the box-shadow to show up